src/Config/SearchEngine/ElasticSuite.php

```diff
@@ -66,7 +66,11 @@ public function get(): array
     {
         $envConfig = (array)$this->stageConfig->get(DeployInterface::VAR_ELASTICSUITE_CONFIGURATION);
 
-        return $this->configMerger->mergeConfigs($this->getConfig(), $envConfig);
+        if ($envConfig && !$this->configMerger->isMergeRequired($envConfig)) {
```

Comment:
In cases when `$envConfig` contains only `{_merge: true}` this method will return an empty array as `isMergeRequired` will return `false`.
You must heck if `$envConfig` isn't empty using method from the `ConfigMerger` class.
`if (!$this->configMerger->isEmpty($envConfig) && !$this->configMerger->isMergeRequired($envConfig)`

Reply:
Fixed